Tools/clinic/clinic.py

```diff
@@ -1249,6 +1250,22 @@ def parser_body(
             parser_prototype = self.PARSER_PROTOTYPE_VARARGS
             parser_definition = parser_body(parser_prototype, '    {option_group_parsing}')
 
+        elif not requires_defining_class and pos_only == len(parameters) - pseudo_args and clinic.limited_capi:
+            # positional-only for the limited C API
+            flags = "METH_VARARGS"
+
+            parser_prototype = self.PARSER_PROTOTYPE_VARARGS
+            parser_code = [normalize_snippet("""
+                if (!PyArg_ParseTuple(args, "{format_units}:{name}",
+                    {parse_arguments}))
+                    goto exit;
+            """, indent=4)]
+            argname_fmt = 'args[%d]'
```

Comment:
It is not correct. In best case it is  unused.

Reply:
Since AC and parsing arguments are complex, I propose to implement incrementally the limited C API. I'm not sure that I get your comment, when is it unused? Do you want to propose a fix?